app/models/passenger.rb

```diff
@@ -26,6 +26,10 @@ class Passenger < ApplicationRecord
   belongs_to :mobility_device, optional: true
 
   before_validation do
+    assign_registration_date
+  end
+
+  def assign_registration_date
```

Comment:
My instinct is that this should be a private method. Do you anticipate using it elsewhere?

Reply:
ha no, there just weren't any existing private methods so I forgot they were a thing.